### DISCUSSION TITLE
output file number

### DISCUSSION THREAD

**Discussion by SmithSimmon** [NCAR/hrldas#199] [Q&A]
Hello, everyone:
I excuted my simulation for 30 years with the  SPINUP_LOOPS=4. I tried to output the variables monthly and put 12 values in a file. However, after running the model for 70 years, no output files were generated. I attempted to change the output interval to daily, but after running for 70 years, no output files were generated either. The model didnt crashed after 70 years and the restart file  generated nomorally. Does anyone encounter the same issue? Any suggestions would be appreciated

**Comment by cenlinhe** [comment]
Can you copy and paste your namelist.hrldas here? Also, are you using the default model code or did you make any code changes by yourself?

**Reply by SmithSimmon** [reply]
Yes, I am using the default model code, just comment out some variables  i dont need to output.
necdf version:4.6.1
hdf5 version: 1.10.1
here is the namelist.hrldas. After the model has been running for 69 years, it stops generating any output files.

 HRLDAS_SETUP_FILE = "/home/wink/forcing/CHINA/results/HRLDAS_setup_2000010100_d1"
 INDIR = "/home/wink/forcing/CHINA/results/data"
 OUTDIR = "/home/wink/forcing/CHINA/output/official_run"

START_YEAR = 1980
START_MONTH = 01
START_DAY = 01
START_HOUR = 00
START_MIN = 00

RESTART_FILENAME_REQUESTED = "/home/wink/forcing/CHINA/results/22/RESTART.1980010100.DOMAIN1"

KDAY = 10958
SPINUP_LOOPS = 4

FORCING_NAME_T = "T2D"
FORCING_NAME_Q = "Q2D"
FORCING_NAME_U = "U2D"
FORCING_NAME_V = "V2D"
FORCING_NAME_P = "PSFC"
FORCING_NAME_LW = "LWDOWN"
FORCING_NAME_SW = "SWDOWN"
FORCING_NAME_PR = "RAINRATE"

DYNAMIC_VEG_OPTION = 2
CANOPY_STOMATAL_RESISTANCE_OPTION = 1
BTR_OPTION = 1
SURFACE_RUNOFF_OPTION = 1
SUBSURFACE_RUNOFF_OPTION = 1
DVIC_INFILTRATION_OPTION = 1
SURFACE_DRAG_OPTION = 1
FROZEN_SOIL_OPTION = 1
SUPERCOOLED_WATER_OPTION = 1
RADIATIVE_TRANSFER_OPTION = 3
SNOW_ALBEDO_OPTION = 2
PCP_PARTITION_OPTION = 1
SNOW_THERMAL_CONDUCTIVITY = 1
TBOT_OPTION = 1
TEMP_TIME_SCHEME_OPTION = 1
GLACIER_OPTION = 1
SURFACE_RESISTANCE_OPTION = 1
SOIL_DATA_OPTION = 1
PEDOTRANSFER_OPTION = 1
CROP_OPTION = 1
IRRIGATION_OPTION = 0
IRRIGATION_METHOD = 0
TILE_DRAINAGE_OPTION = 0

FORCING_TIMESTEP = 86400
NOAH_TIMESTEP = 10800
OUTPUT_TIMESTEP = 2628000    ! 86400

SPLIT_OUTPUT_COUNT = 12    ! or 365 for daily
SKIP_FIRST_OUTPUT = .true.
RESTART_FREQUENCY_HOURS = 262992

! XSTART = 20
! XEND = 20
! YSTART = 203
! YEND = 203

NSOIL=4
soil_thick_input(1) = 0.10
soil_thick_input(2) = 0.30
soil_thick_input(3) = 0.60
soil_thick_input(4) = 1.00

ZLVL = 10.0

SF_URBAN_PHYSICS = 0
USE_WUDAPT_LCZ = 0

**Reply by cenlinhe** [reply]
You OUTPUT_TIMESTEP = 2628000 is not an integer multiplier of 86400 (i.e., 1 day). 2628000/86400 = 30.41667 days
Please set the OUTPUT_TIMESTEP to some other values where it is an integer multiplier of 86400 (i.e., n days).
